### PR TITLE
Feature/rebranding form

### PR DESCRIPTION
**Pull request by Sodik**
*feat TextField and PasswordField "secondary" variant and "large" size
*feat Label variant property
*feat Button new color and size
*feat gradients

### CODE REVIEW

File: packages/ui/src/Button.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import styles from './Button.module.scss'
 export type ButtonKind = 'primary' | 'secondary' | 'danger' | 'transparent'
 
 export type ButtonVariant = 'contained' | 'outlined' | 'text'
-export type ButtonColor = 'primary' | 'secondary' | 'warning' | 'brand'
+export type ButtonColor = 'primary' | 'secondary' | 'warning' | 'brand' | 'brandCustom4'
```
Comment (fantkolja) [inline packages/ui/src/Button.tsx:17]: We need a better naming
How about `"authPrimary"`?

File: packages/ui/styles/constants/_colors.scss [hunk 2]
```diff
@@ -26,3 +26,5 @@ $colorAccessibilityOutline: #9648bd !default;
 $colorBrand: #00bac6 !default;
 $colorShadow: rgb(53 61 73 / 20%) !default;
 $colorMenuDark: #333 !default;
+$colorCustom4: #c6ff3a !default;
+$colorCustom7: #061827 !default;
```
Comment (fantkolja) [inline packages/ui/styles/constants/_colors.scss:30]: It's our new main color for text, so please just redefine `$colorText`

**Review by fantkolja — CHANGES_REQUESTED** [review]

File: packages/ui/styles/constants/_colors.scss (hunk 2 above)
Comment (Sodik) [reply packages/ui/styles/constants/_colors.scss:30]: done

File: packages/ui/src/Button.tsx (hunk 1 above)
Comment (Sodik) [reply packages/ui/src/Button.tsx:17]: done

**Review by utkukaratas — APPROVED** [review]

**Pull request merged**